requirements.txt

```diff
@@ -5,16 +5,17 @@
 # this software in either electronic or hard copy form.
 #
 # List of Python packages needed by this stack
+
+# Needed to run the Linux service.
+daemonize==2.4.7
+
 # Python Jira api.
 # Our fork provides a few fixes that are not part of the official API. Unfortunately
 # contributing back is going to be difficult because Python 2 is not supported anymore.
-git+https://github.com/shotgunsoftware/jira.git@2.0.0.sg.2#egg=jira
+git+https://github.com/shotgunsoftware/jira.git@2.0.0.sg.3#egg=jira
```

Comment:
Updated to use a new patched version of the Jira API that fixes the equality check between Users. This was breaking the tests before, and I guess would have been a bug.